src/os/bluestore/BlueStore.cc

```diff
@@ -7964,16 +8072,29 @@ bluestore_deferred_op_t *BlueStore::_get_deferred_op(
 
 void BlueStore::_deferred_queue(TransContext *txc)
 {
-  dout(20) << __func__ << " txc " << txc << " on " << txc->osr << dendl;
+  dout(20) << __func__ << " txc " << txc << " osr " << txc->osr << dendl;
   std::lock_guard<std::mutex> l(deferred_lock);
```

Comment:
I agree, but the locking gets pretty complicated.  I prefer to make that a follow-on change (when we see this as a point of contention)